package.json

```diff
@@ -28,6 +28,7 @@
     "@babel/preset-typescript": "^7.10.4",
     "@hods/plop-pack": "workspace:^0.1.0",
     "@mdx-js/react": "^1.6.5",
+    "@not-govuk/anchor": "^0.2.9",
```

Comment:
Can't remember if this was raised elsewhere but this line is wrong and should be removed.

Reply:
Seems to be the same as other comments raised above - Don't know how this happened but was likely due to trying to `npm install` a component from elsewhere. Have amended this now